### DISCUSSION TITLE
Daily 10/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#253] [Daily Sprint]
- Qué hice ayer?
- Qué haré hoy
- Dificultades o Impedimentos

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve leyendo los apuntes del Ispc con los temas para este sprint (Enlace de Datos, Directivas, Pipes e Intro a Servicios). Por la noche tuvimos una reunión en donde repasamos los objetivos para este Sprint y vimos como se crean los Servicios.
Hoy continuaré con la segunda parte y trataré de completar la tarea de Servicios.
Por la tarde estaré ocupandome de otro tema, así que no sé si llegaré a terminar todo.

**Comment by rpgrca** [comment]
Hoy voy a revisar las tareas que hay y decidir qué puedo hacer. También voy a probar instalar Django a ver qué tan fácil es.

**Comment by BraianTroncoso** [comment]
Hoy tengo clases de Java, también voy a revisar el código y ver si puedo realizar la parte de servicios de la cual ya me asigné la de carrito.

**Comment by Brian-Almada** [comment]
Hoy voy a intentar hacer la tarea de servicios que me asigné y a estudiar un poco de Angular del nuevo apunte. También intentar avanzar con la tarea pendiente de catálogo.
